solutions/bronze/usaco-832.mdx

```diff
@@ -34,49 +211,53 @@ fixed = [list(map(int, data[i + 2].split(' '))) for i in range(k)]
 cows = [0 for i in range(n)]
 
 for i in range(k):
-  	cows[fixed[i][1] - 1] = fixed[i][0]
```

Comment:
what's w/ the python code changes

Reply:
the code didnt match up with the editorial
i switched the block labeled "case 2" with the block labeled "case 3"